app/models/source_region.rb

```diff
@@ -0,0 +1,7 @@
+require "archived_concern"
+
+class SourceRegion < ApplicationRecord
+  include ArchivedConcern
+
+  belongs_to :source
```

Comment:
These should belong to `:tenant` and `Tenant` should have many regions and subscriptions

Reply:
done